doc/stdlib/base.rst

```diff
@@ -156,6 +156,10 @@ Getting Around
 
    This is only needed if your module depends on a file that is not used via ``include``\ . It has no effect outside of compilation.
 
+.. function:: apropos(string)
+
+   .. Docstring generated from Julia source
+
```

Comment:
Also I think we are only looking in the meta for `Base` so documenting it inline in a different module may not work.